### PR TITLE
feat: tooltip canvas to html

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

#### custom tooltip

```js
const options = {
  tooltip: {
      template: (model: TooltipModel, { header, body }: DefaultTooltipTemplate) => {
        return `
          <div style="display: flex; flex-direction: column; background: black; color: #fff;">
            <p style="padding: 5px; margin: 0; border-bottom: 1px solid #ddd;font-size: 13px; text-align: center">🎊 ${model.category} 🎊</p>
            ${body}
          </div>`;
      },
    },
}
```

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
[08/20] 리뷰완료
고생하셨습니다!

File: src/component/tooltip.ts [hunk 1]
```diff
@@ -1,83 +1,145 @@
 import Component from './component';
```
Comment (jung-han) [inline src/component/tooltip.ts:1]: @shiren tooltip 컴퍼넌트 부분입니다.

File: src/component/tooltip.ts [hunk 2]
```diff
@@ -1,83 +1,145 @@
...
+      header: this.getHeaderTemplate(model),
+      body: this.getBodyTemplate(model),
+    });
+    this.setTooltipPosition(model);
+  }
+
+  getDefaultTemplate(model: TooltipModel, { header, body }: DefaultTooltipTemplate) {
+    return `
+      <div class="tooltip">${header}${body}</div>
+    `;
```
Comment (jungeun-cho) [inline src/component/tooltip.ts:100]: 간단해서 개행하지 않아도 될것같아요.

**Review by jungeun-cho — APPROVED** [review]
[8/20] 이견없습니다. 수고하셨어요~ 👍

**Pull request merged**